go/service/main.go

```diff
@@ -547,6 +548,25 @@ func (d *Service) runHomePoller(ctx context.Context) {
 	return
 }
 
+func (d *Service) runMerkleAudit(ctx context.Context) {
+	if !libkb.IsMobilePlatform() {
```

Comment:
looks like a bug here, it should bail out when `libkb.IsMobilePlatform()`

Reply:
Oops, fixed.

Reply:
kk thanks, trying this again